guides/common/modules/con_troubleshooting-ruby.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Troubleshooting_Ruby_{context}"]
+= Troubleshooting Ruby
+
+If the module fails to enable, it can mean an incorrect module is enabled.
+In that case, you have to resolve dependencies manually as follows:
+
+List the enabled modules:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf module list --enabled
+----
```

Comment:
I think that this paragraph and step has to be included in the postgresql module as well.

Reply:
Sure. I will add it in pgsql file :+1: